### PR TITLE
Include Backround steps before every Scenario

### PR DESCRIPTION
**Pull request by cbandy**

### CODE REVIEW

**Comment by ptrimble** [comment]
Can you also update the README to remove the TODO. Remove the entire section now that we listing these things in the issues.

File: gocumber.go [hunk 1]
```diff
@@ -109,9 +111,7 @@ func (defs *Definitions) Run(t testingFramework, file string) {
 					steps = append(steps, step)
 				})
 			default:
-				for _, step := range scenario.Steps() {
-					steps = append(steps, step)
-				}
+				steps = append(steps, scenario.Steps()...)
```
Comment (ptrimble) [inline gocumber.go]: Just so I understand, the `...` is a fancy way to perform the iteration?

**Comment by ptrimble** [comment]
@cbandy Two nitpick/questions. Looks great. :ship:

File: gocumber.go (hunk 1 above)
Comment (cbandy) [inline gocumber.go]: https://golang.org/ref/spec#Passing_arguments_to_..._parameters

Comment (ptrimble) [inline gocumber.go]: Cool! Thanks.

**Pull request merged**